utils/prometheus/README.md

```diff
@@ -0,0 +1,555 @@
+# Substrate Prometheus Node Exporter
+![grants](./photo_2019-12-13_16-32-53.jpg)
+## Introduction
+
+Prometheus is one of the most widely used monitoring tool for managing high availability services supported by [Cloud Native Computing Foundation](https://www.cncf.io/). By providing Prometheus exporter in substrate, node operators can easily adopt widely used display/alert tool such as Grafana without seting-up/operating external Prometheus agent through RPC connections. Easy access to such monitoring tools will benefit parachain develepers/operators and validators to have much higher availability quality of their services.
+
+## List of Contents
+
+Hack Prometheus in Substrate
+ - Prometheus starter
+ - CLI Config
+ - Metrics Add
+
+Metrics
+ - List of available metrics
+
+Start Prometheus
+ - Install prometheus
+ - Edit Prometheus config file
+ - Start Prometheus
+
+Start Grafana
+ - Install Grafana
+
+## Substrate Dev hack
+### Prometheus starter
+
+Here is the entry point of prometheus core module in Parity Substrate.
+
+In existing sources, refer to the grapana source due to the issue of the wasm.
```

Comment:
Can you add some more details here, I am not sure I follow the intention of the comment.